core/dbt/events/functions.py

```diff
@@ -110,15 +110,14 @@ def cleanup_event_logger():
     EVENT_MANAGER.callbacks.clear()
 
 
-# The default event manager will not log anything, but some tests run code that
-# generates events, without configuring the event manager, so we create an empty
-# manager here until there is a better testing strategy in place.
+# Since dbt-rpc does not do its own log setup, and since some events can
+# currently fire before logs can be configured by setup_event_logger(), we
+# create a default configuration with default settings and no file output.
 EVENT_MANAGER: EventManager = EventManager()
+EVENT_MANAGER.add_logger(
+    _get_logbook_log_config() if flags.ENABLE_LEGACY_LOGGER else _get_stdout_config()
+)
```

Comment:
[answered [here](https://github.com/dbt-labs/dbt-core/pull/6447#issuecomment-1352278750)] Is there a risk of double-adding loggers downstream, leading to extraneous calls to logger.write?